pkg/controller/siddhiprocess/operator.go

```diff
@@ -195,7 +195,7 @@ func (rsp *ReconcileSiddhiProcess) deployParser(sp *siddhiv1alpha2.SiddhiProcess
 		return
 	}
 
-	url := configs.ParserHTTP + sp.Name + "." + sp.Namespace + configs.ParserHealth
+	url := configs.ParserHTTP + sp.Name + ParserExtension + "." + sp.Namespace + configs.ParserHealth
```

Comment:
You can use `url` package or `path` package to generate paths

Reply:
This is a kind of static URL such as 

```
http://appname-parser.default.svc.cluster.local:9090/siddhi-parser/parse
```

 And the dynamically configurable parts are connected with `.` rather than `/` in a generic URL. Thus I think using `url` or `path` packages does not give any advantage.